Formula/m/mysql-client.rb

```diff
@@ -40,6 +40,11 @@ class MysqlClient < Formula
     fails_with :clang
   end
 
+  on_linux do
+    depends_on "krb5"
+    depends_on "libtirpc"
```

Comment:
no linkage for this, maybe just a build dep?

Reply:
I'll run it again and check the error, I think it failed the pre-build dep checks.